rocks/barbican-consolidated/README.md

```diff
@@ -17,16 +17,16 @@ it will help ensure that all layers of the image are imported
 into docker (this is just the top layer).
 
 ```bash
-> skopeo --insecure-policy copy oci-archive:barbican-consolidated_2024.1_amd64.rock docker-daemon:barbican-consolidated:2024.1
+> skopeo --insecure-policy copy oci-archive:barbican-consolidated_2025.1_amd64.rock docker-daemon:barbican-consolidated:2025.1
 ```
 
 If you are interested in giving it a go in Microk8s, you can
 export the image from your docker registry and then into the
 microk8s registry:
 
 ```bash
-> docker save barbican-consolidated:2024.1 > ./barbican-consolidated_2024.1.tar
-> microk8s ctr image import ./barbican-consolidated_2024.1.tar
+> docker save barbican-consolidated:2025.1 > ./barbican-consolidated_2025.1.tar
+> microk8s ctr image import ./barbican-consolidated_2025.1.tar
 # Try with sunbeam
-> juju attach-resource barbican-consolidated barbican-consolidated-image=barbican-consolidated:2024.1
+> juju attach-resource barbican-consolidated barbican-consolidated-image=barbican-consolidated:2025.1
```

Comment:
should be barbican-api-image and another juju attach-resource for barbican-worker-image
This should be updated in stable/2024.1 branch as well

Reply:
I think this should be part of another change that we can backport to stable/2024.1